nilearn/maskers/surface_labels_masker.py

```diff
@@ -47,6 +73,11 @@ class SurfaceLabelsMasker(TransformerMixin, CacheMixin, BaseEstimator):
             This value must be consistent with label values
             and image provided.
 
+    mask_img : :obj:`~nilearn.surface.SurfaceImage` object, optional
+        Mask to apply to labels_img before extracting signals. Defines the \
+        overall area of the brain to consider. The data for each \
+        hemisphere is of shape (n_vertices/2, n_regions).
```

Comment:
```suggestion
        hemisphere is of shape (n_vertices/2, n_regions).
```

not common but I don't think we should assume each hemisphere has the same number of vertices